app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -173,28 +154,30 @@ fun SettingsScreen(modifier: Modifier = Modifier, viewModel: SettingsViewModel =
 fun SettingGroupLabel(text: String, modifier: Modifier = Modifier) {
     Text(
         text = text,
-        modifier = modifier.semantics { heading() },
+        modifier = modifier
+            .semantics { heading() }
+            .padding(horizontal = 8.dp),
         color = MaterialTheme.colorScheme.primary,
         style = MaterialTheme.typography.labelLarge,
     )
 }
 
 @Composable
-fun Setting(
+private fun Setting(
```

Comment:
nit: This and other no-op changes like it are fine for this PR, but for future reference, it would be best to avoid changes outside of the declared commit scope since they make it more difficult to tell what changes are relevant when reviewing.